install.json

```diff
@@ -0,0 +1,5 @@
+{
+  "packageManager": "python",
+  "packageName": "jupyterlab_latex",
```

Comment:
we've been generally tending towards the canonical `-` form, e.g. `jupyterlab-latex`, as that what shows up in PyPI.